src/formatDate.js

```diff
@@ -1,14 +1,41 @@
 'use strict';
 
-/**
+/*
  * @param {string} date
  * @param {string[]} fromFormat
  * @param {string[]} toFormat
  *
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // split date by 4 position('-') to new array;
+  const dateArr = date.split(fromFormat[3]);
+  const receivedDate = {}; // create an empty object
+  const changedDate = [];
+
+  // check each part of date(YY or YYYY) and saving year value to receivedDate;
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        receivedDate['YY'] = dateArr[i].slice(2);
+        receivedDate['YYYY'] = dateArr[i];
```

Comment:
Let's move YY and YYYY into constants and reuse

Reply:
Thank you, that's a good advice! I've change it